src/cpuprofiler.js

```diff
@@ -15,13 +15,17 @@
 // That doesn't work for Chrome in turn, so need to resort to user agent
```

Comment:
I wonder if this is still true today?   Maybe we could remove this whole thing.. but it would require a little testing by someone who uses safari.

@brendandahl would you be interesting it trying to verify if this is still a real issue?

Reply:
I can't help on that (I don't have Safari) ^^

Fixed the other issue

Reply:
Yeah, I'll check it out a little later today.